tests/metadata-handler.js

```diff
@@ -136,6 +136,12 @@ describe('Metadata Handler', () => {
                 expect(intPart.length).to.be.at.most(3);
                 expect(decimalPart.length).to.be.at.most(2);
             });
+
+            it('should default to no digits after the decimal', () => {
```

Comment:
@jcsmorais I know you'll have an opinion about the `it` sentence; you always do 😄